### ISSUE TITLE
Access app with my root domain

### ISSUE THREAD

**Issue by deathwebo** [caprover/caprover#442]
**Hello and thanks for this awesome tool!**

What is the problem?
I want to access my app through my root domain, same I used to point to my DO IP.

Steps to reproduce the problem:

* I have a domain name (Ex: mydomain.com) I use to point to my caprover DO IP.  
* I've created a sample app named `www`. I can access my app as `www.mydomain.com`
* I can't add `mydomain.com` to my app with the `Connect New Domain` option. I get the following error 
<img width="281" alt="Screen Shot 2019-06-08 at 1 17 49 PM" src="https://user-images.githubusercontent.com/8619982/59151922-ec877480-89ef-11e9-8038-d630480b69dd.png">

Answers to the following questions where applicable:
Standard $5 digital ocean droplet

**Comment by githubsaturn** [comment]
What will you see when you enter `yourdomain.com` in your browser.

A) you see the default CapRover page of "Nothing here yet :/".
If this is the case, just a wait 24hr and retry. This will most likely solve the cache problem. 


B) you don't see the default CapRover page, and you see a Chrome error.
In this case, it means that you have not set up your DNS for `yourdomain.com` set up your DNS such that `yourdomain.com` points to the IP address of the server.

DNS A Record
host: @ or empty
IP: IP of the server

**Comment by deathwebo** [comment]
When I enter mydomain.com I get redirected to the www app. While I can work with this, I would like to prevent the redirect

**Comment by githubsaturn** [comment]
> When I enter mydomain.com I get redirected to the www app. While I can work with this, I would like to prevent the redirect


Redirect is probably happening at your DNS level! Use this tool to view your DNS entry for `domain.com`
https://mxtoolbox.com/DNSLookup.aspx

If the IP address that you see in the A records is NOT the IP address of your CapRover server, then you need to avoid the redirect at your DNS level. Look for redirect options in your DNS provider panel (GoDaddy, CloudFlare, etc...)

**Comment by deathwebo** [comment]
Thanks @githubsaturn I think in Digital Ocean I had my DNS A Record with `*` as the value of `host`, just changed that to `@` and I'll wait for the changes to propagate and see if this fixes the issue.

I'll report back later.

**Comment by deathwebo** [comment]
Ok so now I think I am a little confused. Previous issue was kinda fixed but I nuked my droplet to start over again and get a clearer understanding of what I'm doing, so I'd like to ask your help on what the actual steps to setup caprover using a root domain as my default app instead of using a subdomain.

My setup for the DNS A Record:
host: `@`
IP: My DO server IP address

If I try re-run the setup again `caprover serversetup` I get the following:
```
Cannot verify that http://captain.mydomain.com points to your server IP.

Are you sure that you set *.mydomain.com points to MY_DO_IP

Double check your DNS. If everything looks correct, note that, DNS changes take up to 24 hrs to work properly. Check with your Domain Provider.

Error Code: 1107  Message:  Verification Failed.
```

**Comment by githubsaturn** [comment]
Other than `@` record, you need to add another DNS records:
host: `*`
IP: My DO server IP address

**Comment by githubsaturn** [comment]
Use this tool: https://mxtoolbox.com/DNSLookup.aspx

Test the following domains:
```
domain.com
random.domain.com
random2.random.domain.com
```

If all of these domains resolve to IP address of the DigitalOcean droplet, you have your DNS set up correctly. Otherwise, you have DNS problems.

**Comment by deathwebo** [comment]
Thanks, I’ll wait a while for the changes to propagate and report later

**Comment by deathwebo** [comment]
Closing this as I finally got my question solved.

Just for future reference, the steps are kinda like this:

* Follow get started instructions from the docs, using the domain you own.
* After that, add another A record like @githubsaturn explained above, if you're using Digital Ocean then it'd be something like:
```
host: @
IP: Your DO droplet IP
```
* In your app config in rover, add `yourdomain.com` to it and it should work as expected

Thanks a lot for the details and information!
